versions.yaml

```diff
@@ -251,6 +256,11 @@ externals:
     url: "https://github.com/seccomp/libseccomp"
     version: "2.5.4"
 
+  pause:
+    description: "Kubernetes pause container image"
+    repo: "docker://k8s.gcr.io/pause"
+    version: "3.6"
+
```

Comment:
Why do we need the pause image at this point?

Reply:
Removed.

Reply:
Removed.